library/tomee

```diff
@@ -1,67 +1,49 @@
-Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore)
+Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore), Rod Jenkins <jenkir14@nationwide.com> (@scriptmonkey)
 GitRepo: https://github.com/tomitribe/docker-tomee.git
-GitCommit: fbe01211d21051406ffaae5bdf41d1f629caf58f
+GitCommit: 042b7dbd684f9b3649b6d1b4b924f38dedf85c19
```

Comment:
It looks like this needs to be [`858dd76525856a22c57582a40fa1e1607096f056`](https://github.com/tomitribe/docker-tomee/commit/858dd76525856a22c57582a40fa1e1607096f056) or newer to make the Dockerfile ENV versions match the versions here.

Reply:
done